apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -32,76 +34,94 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
             displayName: team.displayName || '',
             interestGroupLeadershipRoleCount: getUniqueIdCount(
               team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                (item) =>
-                  item?.linkedFrom?.usersCollection?.items
-                    .filter(filterOutAlumni)
-                    .flatMap(flattenInterestGroupLeaders),
+                (teamMembershipItem) =>
+                  teamMembershipItem?.linkedFrom?.usersCollection?.items
+                    .flatMap(flattenInterestGroupLeaders)
+                    .filter(
+                      (item) =>
+                        item.interestGroupActive && item.userIsAlumni === false,
+                    )
+                    .map((item) => item.interestGroupId),
               ) || [],
             ),
             interestGroupMemberCount:
               (!team.inactiveSince &&
                 getUniqueIdCount([
                   ...(team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                    (item) =>
-                      item?.linkedFrom?.usersCollection?.items
-                        .filter(filterOutAlumni)
-                        .flatMap(flattenInterestGroupLeaders),
-                  ) || []),
-                  ...(team.linkedFrom?.interestGroupsCollection?.items.flatMap(
-                    (interestGroup) => interestGroup?.sys.id,
+                    (teamMembershipItem) =>
+                      teamMembershipItem?.linkedFrom?.usersCollection?.items
+                        .flatMap(flattenInterestGroupLeaders)
+                        .filter(
+                          (item) =>
+                            item.interestGroupActive &&
+                            item.userIsAlumni === false,
+                        )
+                        .map((item) => item.interestGroupId),
                   ) || []),
+                  ...(team.linkedFrom?.interestGroupsCollection?.items
+                    .filter((interestGroup) => !!interestGroup?.active)
+                    .flatMap((interestGroup) => interestGroup?.sys.id) || []),
                 ])) ||
               0,
             interestGroupPreviousLeadershipRoleCount: getUniqueIdCount(
               team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                (item) =>
-                  item?.linkedFrom?.usersCollection?.items
-                    .filter(filterAlumni)
-                    .flatMap(flattenInterestGroupLeaders),
+                (teamMembershipItem) =>
```

Comment:
I believe the `!team.inactiveSince` verification is also missing here. I'll point it out in some of the tests